packages/user-interface/src/components/ExtraCaseDetails.tsx

```diff
@@ -0,0 +1,151 @@
+import { ReactNode } from "react";
+import { DescriptionList } from "@gemeente-denhaag/descriptionlist";
+import TableList from "./TableList";
+import UnorderedList, {
+  UnorderedListItem,
+} from "@gemeente-denhaag/unorderedlist";
+import SectionHeader from "./SectionHeader";
+import { FormattedMessage } from "react-intl";
+import { CellObject } from "./Table";
+
+interface Props {
+  data: Details[];
+}
+
+export interface Extrazaakdetails {
+  data: Details[];
+  zaak?: string;
+}
+
+export interface Table {
+  headers: Details[];
+  rows: Details[][];
+}
+
+export type Waarde = string | boolean | string[] | Details[] | Table;
+
+export interface Details {
+  key?: string;
+  type?: "bold" | "italic" | "date" | "table";
+  waarde: Waarde;
+  heading?: string;
+  omschrijving?: string;
+}
+
+const hasDetailsWaardes = (waarde: Waarde) =>
+  Array.isArray(waarde) && waarde.every((item) => typeof item === "object");
+const hasStringWaardes = (waarde: Waarde) =>
```

Comment:
Nog handig om deze onder utils onder te brengen?

Reply:
Dit zijn echt specifieke dingen voor die extra zaak details, ik vraag mij af of dit hergebruikt zou gaan worden.